dts/arm/microchip/mec1501hsz.dtsi

```diff
@@ -419,9 +419,11 @@
 			compatible = "microchip,xec-adc";
 			reg = <0x40007c00 0x90>;
 			interrupts = <78 0>, <79 0>;
+			girqs = <17 8>, <17 9>;
+			pcrs = <3 3>;
 			status = "disabled";
 			#io-channel-cells = <1>;
-			clktime = <32>;
```

Comment:
@Manimaran-A elaborate on this change, we observed in the past an inaccuracy on ADC readings with wrong clktime. Refer to 
https://github.com/zephyrproject-rtos/zephyr/commit/e4481f82cb02afced321adc845f771e82a541e6e

Reply:
cc @AdiIntel

Reply:
Before the driver merge, I observed that MEC172x driver did not explicitly initialize the config_reg. So it  uses the default value that is 1. But in MEC1501, i we configured it with 32 (through dts). 

So, when i did the MEC172x and MEC15xx merge thought of bring both as same configuration value and kept 1 in the dts.

I will update the value back to 32 for MEC1501.

Reply:
clktime better be set minimum 32.

Just to share our learning, we couldn't get appropriate ADC values with MEC1501 evb card with default clktime ( = 1). Readings were off by ~30%.
Increasing this clktime value was only the solution to get better accuracy even though it adds a bit of latency.
I noticed the same issue on boards using 17xx - where readings are in-accurate, and now I know why. :) - this new driver still had default ( =1) value for clktime. Agree with you to update back to 32.

Reply:
Ok, i revert the configuration back to 32